clients/drcachesim/CMakeLists.txt

```diff
@@ -60,6 +60,8 @@ elseif (AARCHXX)
   else ()
```

Comment:
Like Stanislaw pointed out in his comment: about the PR description and title: please follow https://dynamorio.org/page_code_reviews.html.

The title and description should be something like
```
i#5781: fix RISCV64 compilation

Fixes a compilation in the RISCV64 build due to <insert-details-here>

Fixes: #5781
```
For this PR since I need to commit it on your behalf anyway (since you don't have write access currently), I'll make the changes in the final commit message.

Reply:
I will remember this experience as a reference for next time.
Thank you very much. @abhinav92003